metrics/docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: "3.9"
+services:
+  influxdb:
+    image: "influxdb:1.8"
```

Comment:
Why used image of version `1.8`? I'm ask cuz looks like exist also `1.9` and `2.0`:
* https://hub.docker.com/_/influxdb
* https://docs.influxdata.com/enterprise_influxdb/v1.9/query_language/explore-schema/
* https://docs.influxdata.com/influxdb/v2.0/query-data/

Reply:
v1.8 is officially supported OSS version
v1.9 - is so called "enterprise" version.
v2.0 - [doesn't support graphite protocol](https://docs.influxdata.com/influxdb/v2.0/upgrade/v1-to-v2/automatic-upgrade/#supported-protocols) out of the box

Reply:
Clear.